src/docs/cookbook/lists/floating-app-bar.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Place a floating app bar above a list 
+prev:
+  title: Creating lists with different types of items
+  path: /docs/cookbook/lists/mixed-list
+next:
+  title: Working with long lists
+  path: /docs/cookbook/lists/long-lists
+---
+
+To make it easier for users to view a list of items, you may want to hide the
+app bar as the user scrolls down the list. This is especially true if your app
+displays a "tall" app bar that occupies a lot of vertical space.
+
+Traditionally, you create an app bar by providing an `appBar` property to the
+`Scaffold` Widget. This will create a fixed app bar that always remains above
+the `body` of the `Scaffold`.
+
+If you'd like to create a floating app bar scrolls offscreen as the user scrolls
+through a list, you need to move the app bar from the Scaffold into a
+[`CustomScrollView`](https://docs.flutter.io/flutter/widgets/CustomScrollView-class.html).
+This allows you to create an app bar that scrolls offscreen as you scroll
+through a list of items contained inside the `CustomScrollView`!
```

Comment:
This feels a little repetitive with the first sentence. How about:

Moving the app bar from the Scaffold into the [`CustomScrollView`][] allows you to create an
app bar that scrolls offscreen as you scroll through a list of items contained inside the
`CustomScrollView`.